docs/topics/api/addons.rst

```diff
@@ -90,19 +91,22 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json string current_version.files[].url: The (absolute) URL to download a file.
     :>json string default_locale: The add-on default locale for translations.
     :>json string|object|null description: The add-on description.
-    :>json string edit_url: The URL to the developer edit page for this add-on.
+    :>json string edit_url: The URL to the developer edit page for the add-on.
     :>json string guid: The add-on `extension identifier <https://developer.mozilla.org/en-US/Add-ons/Install_Manifests#id>`_.
     :>json string|object|null homepage: The add-on homepage.
-    :>json string icon_url: The URL to icon for this add-on (cache-busting query string included).
+    :>json string icon_url: The URL to icon for the add-on (including a cachebusting query string).
```

Comment:
I googled the two and got more results with "cachebusting", so I went with that :)